elfo-core/src/actor_status.rs

```diff
@@ -23,13 +23,22 @@ impl ActorStatus {
     pub(crate) const TERMINATED: ActorStatus = ActorStatus::new(ActorStatusKind::Terminated);
     pub const TERMINATING: ActorStatus = ActorStatus::new(ActorStatusKind::Terminating);
 
+    #[cfg(not(feature = "test-util"))]
     const fn new(kind: ActorStatusKind) -> Self {
         Self {
             kind,
             details: None,
         }
     }
 
+    #[cfg(feature = "test-util")]
+    pub const fn new(kind: ActorStatusKind) -> Self {
```

Comment:
What is the reason for this change? Is it to create `FAILED` and `TERMINATED` statuses in tests?

Reply:
yes

Reply:
I think it's more consistent to extend the visibility of `ActorStatus::X` costs under `test-util` instead of providing another public way to create `ActorStatus.`

Reply:
fixed